packages/astro/src/template/4xx.ts

```diff
@@ -1,4 +1,4 @@
-import { encode } from 'html-entities';
+import { escape } from 'html-escaper';
```

Comment:
`html-entities` and `html-escaper` do the same thing. Since we use `html-escaper` more, I've swap to it. `html-entities` was only used here once.